biogeochem/EDCohortDynamicsMod.F90

```diff
@@ -1607,7 +1676,7 @@ subroutine copy_cohort( currentCohort,copyc )
   end subroutine copy_cohort
 
   !-------------------------------------------------------------------------------------!
-  function count_cohorts( currentPatch ) result ( backcount )
+  subroutine count_cohorts( currentPatch )
```

Comment:
The subroutine does set a global cohort count, which is used in the photosynthesis loop.  Since this was originally a function, I had overlooked that, thinking that the function's result wasn't even used, and therefore the function isn't used.